src/compile.coffee

```diff
@@ -11,24 +11,22 @@ compile = (mediaType, apiElements, filename) ->
   apiElements.freeze()
 
   transactions = []
-  errors = apiElements.errors.map(compileAnnotation)
-  warnings = apiElements.warnings.map(compileAnnotation)
+  annotations = apiElements.annotations.map(compileAnnotation)
 
   for {httpTransactionElement, exampleNo} in findRelevantTransactions(mediaType, apiElements)
-    {transaction, annotations} = compileTransaction(mediaType, filename, httpTransactionElement, exampleNo)
-    transactions.push(transaction) if transaction
-    errors = errors.concat(annotations.errors)
-    warnings = warnings.concat(annotations.warnings)
+    result = compileTransaction(mediaType, filename, httpTransactionElement, exampleNo)
+    transactions.push(result.transaction) if result.transaction
+    annotations = annotations.concat(result.annotations)
 
-  {mediaType, transactions, errors, warnings}
+  {mediaType, transactions, annotations}
 
 
 compileAnnotation = (annotationElement) ->
   {
+    type: annotationElement.classes.getValue(0)
     component: 'apiDescriptionParser'
-    code: annotationElement.code?.toValue()
     message: annotationElement.toValue()
-    location: annotationElement.sourceMapValue
+    location: annotationElement.sourceMapValue or [[0, 1]]
```

Comment:
Why did you add this default value?

Reply:
Because some annotations seem to be without source map, genuinely. I wanted to keep the behavior consistent, so in case there is no source map, I'm setting "the first character of the first line" as the place of where the problem occurs.